src/API/WorkController.ts

```diff
@@ -30,21 +29,19 @@ export class WorkController {
 
   async getById(id: string): Promise<any> {
     this.logger.trace({ method: 'getById', id }, 'Getting Work by Id from DB')
-    return this.collection.findOne({ id }, { fields: { _id: false } })
+    return this.collection.findOne({ id }, { projection: { _id: false } })
   }
 
-  async getByFilters(worksFilters: WorksFilters = {}): Promise<any> {
+  async getByFilters(worksFilters: WorksFilters = {}): Promise<ReadonlyArray<any>> {
     this.logger.trace({ method: 'getByFilters', worksFilters }, 'Getting Work by Filters from DB')
-    const { offset, limit }: { offset?: string; limit?: string } = omitUndefined(worksFilters)
```

Comment:
Here I didn't really find a reason to use `omitUndefined` to then spread-assign to offset and limit, since those would get default values afterwards, which would override `undefined`.

Reply:
Ok that makes a lot of sense.